src/EnergyPlus/DataGlobalConstants.cc

```diff
@@ -652,6 +659,112 @@ namespace DataGlobalConstants {
         return ResourceTypeChar;
     }
 
+    enum FuelTypeNumber
+    {
+        ELECTRICITY,
+        NATURALGAS,
+        DIESEL,
+        GASOLINE,
+        COAL,
+        FUELOILNO1,
+        FUELOILNO2,
+        PROPANE,
+        OTHERFUEL1,
+        OTHERFUEL2
+    };
+
+    FuelTypeNumber ConvertFueltypeString(std::string FuelTypeInput)
+    {
+        if (FuelTypeInput == "ELECTRICITY")
+            return ELECTRICITY;
+        else if (FuelTypeInput == "NATURALGAS")
+            return NATURALGAS;
+        else if (FuelTypeInput == "DIESEL")
+            return DIESEL;
+        else if (FuelTypeInput == "GASOLINE")
+            return GASOLINE;
+        else if (FuelTypeInput == "COAL")
+            return COAL;
+        else if (FuelTypeInput == "FUELOILNO1")
+            return FUELOILNO1;
+        else if (FuelTypeInput == "FUELOILNO2")
+            return FUELOILNO2;
+        else if (FuelTypeInput == "PROPANE")
+            return PROPANE;
+        else if (FuelTypeInput == "OTHERFUEL1")
+            return OTHERFUEL1;
+        else if (FuelTypeInput == "OTHERFUEL2")
+            return OTHERFUEL2;
+    }
+
+    void ValidateFuelType(std::string FuelTypeInput)
+    {
+        // FUNCTION INFORMATION:
+        //       AUTHOR         Dareum Nam
+        //       DATE WRITTEN   May 2020
+        //       MODIFIED       na
+        //       RE-ENGINEERED  na
```

Comment:
Please delete these comment lines with "na" in a future commit. I don't mind the name/date comments (could be 1 line though) but other disagree. So at the very least get rid of the na lines. Keep new functions as slim as possible.